### PR TITLE
Added API views for Providers, campus, qualifications, campus_events, qualification_events, saqa_qualifications

### LINKED ISSUE #330: set up django-rest-framework

### LINKED ISSUE #333: /campus_event

### LINKED ISSUE #334: /qualification_event

### LINKED ISSUE #336: /qualification

### LINKED ISSUE #337: /campus

### LINKED ISSUE #338: /saqa_qualification

### PR DESCRIPTION
**Pull request by Christiaanvdm**
Just the basic data with no extra/special methods for now.
This closes #330 
This closes #336 
This closes #338 
This closes #333
This closes #334
This closes #337 
I forgot to demo that adding /?format=json to any of the views displays them as JSON.
NewGIF

### CODE REVIEW

File: django_project/api/serializers/provider.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from ford3.models.provider import Provider
+from api.serializers.campus import CampusSerializer
+
+
+class ProviderSerializer(serializers.ModelSerializer):
+    campus = CampusSerializer(many=True)
+
+    class Meta:
+        model = Provider
+        fields = '__all__'
```
Comment (pierrealixt) [inline django_project/api/serializers/provider.py]: surely not all of them (created_by_id, created_at, ...)

File: django_project/api/urls.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        SAQAQualificationViewSet.as_view({'get': 'list'}),
+        name='show-saqa-qualifications-api'),
+
+    url(r'^(?P<version>(v1))/saqa-qualifications/(?P<pk>\d+)/$',
+        SAQAQualificationViewSet.as_view({'get': 'retrieve'}),
+        name='show-saqa-qualification-api'),
+
+    url(r'^(?P<version>(v1))/campus-events/$',
+        CampusEventViewSet.as_view({'get': 'list'}),
+        name='show-campus-events-api'),
```
Comment (pierrealixt) [inline django_project/api/urls.py:53]: `show-campus-events-api` and `show-campus_event-api` are really similar name. why?

**Comment by pierrealixt** [comment]
Very nice!
Does it make sense for the end-user to receive a payload with meaningless identifier (id: 2, 4, 42)?

**Comment by pierrealixt** [comment]
```
REST_FRAMEWORK = {
    'DEFAULT_RENDERER_CLASSES': (
        'rest_framework.renderers.JSONRenderer',
        'rest_framework.renderers.BrowsableAPIRenderer',
    )
}
```

**Comment by Christiaanvdm** [comment]
NewGIF

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**